.buildkite/pipeline.deploy-adapters.yml

```diff
@@ -59,6 +59,28 @@ steps:
     agents:
       queue: nano
 
+  - label: deploy Sierra reader
+    command: |
+      ./sierra_adapter/sierra_reader_new/deploy.sh
+
+    plugins:
+      - wellcomecollection/aws-assume-role#v0.2.2:
+          role: "arn:aws:iam::760097843905:role/platform-ci"
+
+    agents:
+      queue: nano
+
+  - label: deploy Sierra progress reporter
+    command: |
+      ./sierra_adapter/sierra_progress_reporter/deploy.sh
+
+    plugins:
+      - wellcomecollection/aws-assume-role#v0.2.2:
+          role: "arn:aws:iam::760097843905:role/platform-ci"
+
+    agents:
+      queue: nano
```

Comment:
Ideally I'd do this with something like a matrix build (and maybe we will as more stuff goes to Lambda), but for now this is fine.